python/ray/workflow/api.py

```diff
@@ -478,5 +478,67 @@ def delete(workflow_id: str) -> None:
     wf_storage.delete_workflow()
 
 
+@PublicAPI(stability="beta")
+def wait(workflows: List[Workflow],
+         num_returns: int = 1,
+         timeout: Optional[float] = None):
+    """Return a list of workflows that are ready and a list of workflows that
+    are not.
```

Comment:
Hmm actually returns a Workflow that wraps this, not the lists themselves, right?

Reply:
Can you explain how it is implemented, i.e. with a separate workflow.wait step whose output is the ready and unready workflows, etc?